### DISCUSSION TITLE
Hide volume bar option (But still have media controls)

### DISCUSSION THREAD

**Discussion by Auranapse** [ShankarBUS/ModernFlyouts#1104] [Ideas]
Hi! I think it would be a great option to be able to just hide the volume bar from the audio flyout, and still continue to make use of the media controls.

![image](https://user-images.githubusercontent.com/8434126/191854462-f9e9d4c3-e364-4c1f-a237-8c95d2eb9f07.png)

I personally really like the default volume bar introduced with 22H2 and using that along with the media controls would be a perfect setup.

Also, with my current setup, if there's no media controls, I'll be basically be left with this:
![image](https://user-images.githubusercontent.com/8434126/191855124-34b02e67-4075-47b2-8fe9-7b143ea8668a.png)
Having bottom alignment as a option would be great too.

**Comment by etomoinick** [comment]
I think about it after Win11 release. I tried to turn off audio module, but ModernFlyouts crashes every time :(

**Comment by Necr0man** [comment]
If there is a settings file, can someone provide the location to be manually edited and disable the audio module, because of ModernFlyout crashing every time we disable this setting?
Thanks!

**Comment by nothinglastsforever** [comment]
Any progress on this option? Especially given [that this new feature is coming to relocate the position of on-screen indicators](https://x.com/phantomofearth/status/1936057697125769281/) (volume, brightness, etc.) 

![image](https://github.com/user-attachments/assets/1f3c623d-971c-4457-91f5-057bc45ecbf3)

Would also love to be able to keep the media player permanently affixed to the display (with a close button if required).
